spec/reek/cli/command/todo_list_command_spec.rb

```diff
@@ -37,6 +37,40 @@
         result = command.execute
         expect(result).to eq(Reek::CLI::Options::DEFAULT_SUCCESS_EXIT_CODE)
       end
+
+      it 'writes a todo file' do
+        command.execute
+        expected_yaml = { 'FeatureEnvy' => { 'exclude' => ['self'] } }.to_yaml
```

Comment:
That's very confusing here and below. The `expected_yaml` is based on the default values of our factory_girl. But you have no reference here to our factory definitions so an innocent reader will be lost here why this is "expected".
It's also duplicated from our factories - if we change the value in the factory those specs will break which is detrimental to our maintenance. 
How about rather using factory_girls `attributes_for` -> https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#using-factories